### DISCUSSION TITLE
Package gnuplot is required in LabA: Host Code Optimization

### DISCUSSION THREAD

**Discussion by poppin-mice** [bol-edu/HLS-SOC-Discussions#28] [Q&A] [answered]
It seems that gnuplot is required in topic 6 of LabA, host code optimization. However, it requires sudo permission to install the packages. Is it possible to install gnuplot by TAs? Thanks for the emergent help!
![image](https://user-images.githubusercontent.com/11053589/195132840-f91664d6-8f40-4eb3-a640-fdcb84c25038.png)

**Comment by kevinjantw** [comment, marked as answer]
Thanks for the report,
the package was installed please retry this issue.
